Sources/MisticaCommon/Components/GradientView+UIKit.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  GradientView+UIKit.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import SwiftUI
+
+public class GradientSwiftUIViewController: UIViewController {
```

Comment:
I would add a comment here to explain why you should use this hosting VC
```suggestion
/// ...
public class GradientSwiftUIViewController: UIViewController {
```